### PR TITLE
Syndicate turrets will no longer target Cayenne

### LINKED ISSUE #13711: Turrets will fire at simple animals even if they are both in the same faction.
What it says on the tin. The turret will NOT fire at monkeys, humans, xenos, or cyborgs if they're in the same faction, but don't care in the slightest if it's a simple animal and will shoot at those anyway.

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
And respect factions of simple animals in general

Fixes #13711

:cl: Kor
rscadd: Cayenne will no longer be targeted by the syndicate turrets.
/:cl:

### CODE REVIEW

**Comment by Shapsy** [comment]
Fix of the fucking year

**Comment by Bawhoppen** [comment]
no longer aiming through walls though?

File: code/game/machinery/portable_turret.dm [hunk 1]
```diff
@@ -470,8 +470,9 @@
 
 	if(check_anomalies)	//if it's set to check for xenos/simpleanimals
 		for(var/mob/living/simple_animal/SA in turretview)
-			if(!SA.stat && (!SA.has_unlimited_silicon_privilege || !(faction in SA.faction))) //don't target dead animals or NT maint drones.
-				targets += SA
+			if(SA.stat || faction in SA.faction || SA.has_unlimited_silicon_privilege) //don't target dead animals or NT maint drones.
```
Comment (tkdrg) [inline code/game/machinery/portable_turret.dm]: should probably be `(faction in SA.faction)` if I'm getting my priority right

Comment (KorPhaeron) [inline code/game/machinery/portable_turret.dm]: Works currently (doesn't shoot cayenne, shoots normal space carp and stops shooting when it dies) but I can change it

Comment (tkdrg) [inline code/game/machinery/portable_turret.dm]: yeah i was unsure since the ref doesn't list the priority of `in` for some reason

Comment (KorPhaeron) [inline code/game/machinery/portable_turret.dm]: You're the boss

**Pull request merged**